engine/src/main/java/org/terasology/rendering/md5/MD5AnimationLoader.java

```diff
@@ -224,7 +225,7 @@ private void readBounds(BufferedReader reader, MD5 md5) throws IOException {
             max.x = Math.max(a.x, b.x);
             max.y = Math.max(a.y, b.y);
             max.z = Math.max(a.z, b.z);
-            md5.bounds[i] = AABB.createMinMax(min, max);
+            md5.bounds[i] = AABB.createMinMax(JomlUtil.from(min), JomlUtil.from(max));
```

Comment:
The lines above can be simplified using joml methods:
```
Vector3f min = new Vector3f(a).min(b);
Vector3f max = new Vector3f(a).max(b);
```

Reply:
its a list of AABB bounds. unless we want to start using the AABB  bound that JOML already implements. the way it is right now is probably fine.

Reply:
I'm talking about this stuff:
```
Vector3f min = new Vector3f();
min.x = Math.min(a.x, b.x);
min.y = Math.min(a.y, b.y);
min.z = Math.min(a.z, b.z);
Vector3f max = new Vector3f();
max.x = Math.max(a.x, b.x);
max.y = Math.max(a.y, b.y);
max.z = Math.max(a.z, b.z);
```

Reply:
I've been debating if it is really necessary. I think all the md5 stuff is going to go away at some point.